drivers/eks/eks_driver.go

```diff
@@ -733,7 +734,68 @@ func getParameterValueFromOutput(key string, outputs []*cloudformation.Output) s
 func (d *Driver) Update(ctx context.Context, info *types.ClusterInfo, options *types.DriverOptions) (*types.ClusterInfo, error) {
 	logrus.Infof("Starting update")
 
-	// nothing can be updated so just return
+	state, err := getStateFromOptions(options)
+	if err != nil {
+		return nil, fmt.Errorf("error parsing state: %v", err)
+	}
+
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String(state.Region),
+		Credentials: credentials.NewStaticCredentials(
+			state.ClientID,
+			state.ClientSecret,
+			"",
+		),
+	})
+	if err != nil {
+		return nil, fmt.Errorf("error getting new aws session: %v", err)
+	}
+
+	svc := autoscaling.New(sess)
+
+	filterName := "key"
+	filterValue := "kubernetes.io/cluster/" + state.ClusterName
+
+	describeTagsResult, err := svc.DescribeTags(&autoscaling.DescribeTagsInput{
+		Filters: []*autoscaling.Filter{
+			&autoscaling.Filter{
+				Name:   &filterName,
```

Comment:
FYI, the aws package has a func `aws.String( some string )` which returns a string pointer to whatever string you pass in.  I like to use it when creating structs so that I am consistent with the AWS code examples.

Check out this usage of it:
https://github.com/rancher/kontainer-engine/blob/master/drivers/eks/eks_driver.go#L262